Modules/ChequeModule/Views/ChequeView.xaml

```diff
@@ -17,17 +18,160 @@
         </ResourceDictionary>
     </UserControl.Resources>
 
-    <Grid Background="{StaticResource Secondary}">
+    <Grid Background="{StaticResource Secondary}" >
+
         <Grid.RowDefinitions>
             <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
+            <RowDefinition Height="17*" />
+            <RowDefinition Height="331*"/>
+            <RowDefinition Height="15*"/>
+            <RowDefinition Height="60*"/>
+            <RowDefinition Height="23*"/>
         </Grid.RowDefinitions>
         <Grid.ColumnDefinitions>
             <ColumnDefinition Width="auto" />
+            <ColumnDefinition Width="3*"/>
+            <ColumnDefinition Width="55*"/>
+            <ColumnDefinition Width="4*"/>
+            <ColumnDefinition Width="84*"/>
+            <ColumnDefinition Width="4*"/>
         </Grid.ColumnDefinitions>
         <Button
-            Grid.Column="0"
             Command="{Binding NavigateBack}"
-            Style="{StaticResource BackImageButtonStyle}" />
+            Style="{StaticResource BackImageButtonStyle}" HorizontalAlignment="Center" Width="50" />
+
+        <StackPanel Grid.Row="4" Grid.Column="2">
+            <Button 
+                Width="101" 
+                Height="39" 
+                HorizontalAlignment="Left" >Save</Button>
+        </StackPanel>
+        <StackPanel Grid.Row="4" Grid.Column="2">
+            <Button 
+                Width="101" 
+                Height="39" 
+                HorizontalAlignment="Right" >Cancel</Button>
+        </StackPanel>
+
+        <StackPanel Grid.Row="1" Grid.Column="2" Margin="0,15,0,0" Grid.RowSpan="2" >
```

Comment:
The `Margin` property is one of the attribute properties on the customization of the layout size. 
When there is a definition is defined in a Grid, we can use the `VerticalAlignment` or `HorizontalAlignment` to control the position of the UI component under a layout control.

The detail I will checked with you during the meeting.